modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/utils/SW360ReleaseAdapterUtils.java

```diff
@@ -72,18 +74,13 @@ private static void setCPEId(SW360Release release, Artifact artifact) {
                 .ifPresent(release::setCpeId);
     }
 
-    private static void setCoordinates(SW360Release release, Artifact artifact) {
-        release.setCoordinates(getMapOfCoordinates(artifact));
-    }
+    private static void setPackageUrls(SW360Release sw360Release, Artifact artifact) {
+        Set<String> packageURLS = artifact.askForAll(ArtifactCoordinates.class).stream()
+                .map(ArtifactCoordinates::getPurl)
+                .map(PackageURL::toString)
```

Comment:
I am not sure if toString is the right method, the "official" method for getting the String representation of the PURL is canonicalize().

Reply:
While [toString() indeed just calls canonicalize()](https://github.com/package-url/packageurl-java/blob/29a68812366bd12b0541edcb1487e175b98fbaf6/src/main/java/com/github/packageurl/PackageURL.java#L366-L369) I'd also prefer to be explicit and directly call `canonicalize()` here.